third_party/custom-babel-helpers.js

```diff
@@ -0,0 +1,112 @@
+/**
```

Comment:
move into sub directory (called custom-babel-helpers) and add the LICENSE file from the respective project.